packages/babel-preset-mapbox/README.md

```diff
@@ -4,82 +4,124 @@ A configurable Babel preset which abstracts away the most common Babel plugins a
 
 ## Usage
 
-Install `babel-preset-mapbox` by running the following command at the root of your project:
+This package works with **Babel 7 and above**. To install Babel run the following command:
```

Comment:
I don't think we need to include installation instructions for Babel. There are so many different ways to use Babel, it's kind of a rabbit hole.

Reply:
Babel is the rabbit hole.